### PR TITLE
Send html enable/disable flag for candidate sets with spark request

### PR DESCRIPTION
**Pull request by vansika**
We'd want to control HTML generation for candidate sets with spark requests. In this PR I have added the functionality for the same.

### CODE REVIEW

File: listenbrainz/spark/request_manage.py [hunk 1]
```diff
@@ -156,16 +156,18 @@ def request_model(rank, itr, lmbda, alpha):
 @click.option("--days", type=int, default=7, help="Request recommendations to be generated on history of given number of days")
 @click.option("--top", type=int, default=20, help="Calculate given number of top artist.")
 @click.option("--similar", type=int, default=20, help="Calculate given number of similar artist.")
+@click.option("--html", is_flag=False, help='Enable/disable HTML file generation')
```
Comment (paramsingh) [inline listenbrainz/spark/request_manage.py:159]: >is_flag=False

Should this be True? This is a flag, right?

Comment (mayhem) [reply listenbrainz/spark/request_manage.py:159]: beat me to it. :)

Comment (vansika) [reply listenbrainz/spark/request_manage.py:159]: I wanted a flag whose default value should be `False`.

Any other way?

Comment (vansika) [reply listenbrainz/spark/request_manage.py:159]: Maybe click.bool

Comment (mayhem) [reply listenbrainz/spark/request_manage.py:159]: is_flag=True, default=False

should do it. See https://click.palletsprojects.com/en/7.x/options/

Comment (vansika) [reply listenbrainz/spark/request_manage.py:159]: Thanks🌹

**Pull request merged**